FpsCheck.py

```diff
@@ -409,34 +409,35 @@ def find_invalid_alias_eSLDs(self, check_sets):
         """Checks that eSLDs match their alias, and that country codes are 
         members of icann
         Reads the ccTLDs and makes sure that they match their equivalent sites,
-        and that they're domains are part of ICANNS list of country codes.
-        If either of these is not the case, appends an error to the error_list
+        and that their eTLDs are part of ICANN's list of country codes.
+        If either of these is not the case, appends an error to the error_list.
+        Note: A site may list a variant with "com" as its eTLD IFF the site 
+        being aliased has an eTLD on ICANN's list of countrycodes. 
         Args:
-            check_sets: a dictionary of primary->FpsSet
+            check_sets: Dict[string, FpsSet]
         Returns:
             None
         """
         for primary in check_sets:
             curr_set = check_sets[primary]
             if curr_set.ccTLDs:
```

Comment:
```suggestion
            if not curr_set.ccTLDs:
                continue
```

Optional: you could remove one level of indentation by flipping the sense of this if statement.

Reply:
Done